requirements.txt

```diff
@@ -1,9 +1,9 @@
 pytest
```

Comment:
We don't use `pytest-html`, `assertpy`, `selenium`, `pytest-selenium`, or `xmlrunner` so you can remove them.